addons/web_editor/models/ir_ui_view.py

```diff
@@ -270,3 +271,89 @@ def get_related_views(self, key, bundles=False):
         View = self.with_context(active_test=False, lang=None)
         views = View._views_get(key, bundles=bundles)
         return views.filtered(lambda v: not v.groups_id or len(user_groups.intersection(v.groups_id)))
+
+    # --------------------------------------------------------------------------
+    # Snippet saving
+    # --------------------------------------------------------------------------
+
+    @api.model
+    def _get_default_snippet_thumbnail(self, snippet_class=None):
+        return '/web_editor/static/src/img/snippets_thumbs/s_custom_snippet.png'
+
+    @api.model
+    def _get_snippet_addition_view_key(self, template_key, key):
```

Comment:
a method just to concat 2 strings? overkill imo
+ it makes the coder harder to go check what the method does just to see a concat

Reply:
Used at two places, so it seems ok to me. I would have hesitated for two occurences of just a concatenation of two strings but since @fja-odoo did it, seems ok to keep it to me.

Reply:
imo, seems really useless too... if you don't plan to override it later...
In this case you can do function for each line of code.

![image](https://user-images.githubusercontent.com/4848797/72730264-51fc3000-3b91-11ea-8881-b06d4a9afcb1.png)